jabgui/src/main/java/org/jabref/gui/walkthrough/declarative/NodeResolver.java

```diff
@@ -50,12 +54,17 @@ static NodeResolver fxId(@NonNull String fxId) {
     /// @param glyph the graphic of the button
     /// @return a resolver that finds the button by graphic
     static NodeResolver buttonWithGraphic(IconTheme.JabRefIcons glyph) {
-        return scene -> scene.getRoot().lookupAll(".button").stream()
-                             .filter(node -> node instanceof Button button
-                                     && Optional.ofNullable(findNode(button.getGraphic(), JabRefIconView.class::isInstance))
-                                                .map(JabRefIconView.class::cast).filter(icon -> icon.getGlyph() == glyph)
-                                                .isPresent())
-                             .findFirst();
+        return scene -> Streams.concat(scene.getRoot().lookupAll(".button").stream(),
+                                       scene.getRoot().lookupAll(".icon-button").stream())
+                               .filter(node -> {
+                                   if (!(node instanceof ButtonBase button)) {
+                                       return false;
+                                   }
+                                   Node graphic = button.getGraphic();
+                                   return (graphic instanceof JabRefIconView jabRefIconView) && jabRefIconView.getGlyph() == glyph ||
+                                           (graphic instanceof FontIcon fontIcon) && fontIcon.getIconCode() == glyph.getIkon();
+                               })
+                               .findFirst();
```

Comment:
See if this works (and remove Streams import):
```suggestion
        return scene -> scene.getRoot()
                             .lookupAll(".button, .icon-button")
                             .stream()
                             .filter(node -> {
                                 if (!(node instanceof ButtonBase button)) {
                                     return false;
                                 }
                                 Node graphic = button.getGraphic();
                                 return (graphic instanceof JabRefIconView jabRefIconView && jabRefIconView.getGlyph() == glyph) ||
                                         (graphic instanceof FontIcon fontIcon && fontIcon.getIconCode() == glyph.getIkon());
                             })
                             .findFirst();
```

Reply:
I added a comment one line before this. Basically, the lookupAll doesn't support multiple selector